web/index.js

```diff
@@ -41,15 +41,19 @@ function pick({multiple, type})
     {
       body.removeEventListener('focus', onfocus, true);
 
-      const {files} = input;
-      if ( !files.length ) {
-        const error = new Error('User canceled document picker');
-              error.code = E_DOCUMENT_PICKER_CANCELED;
+      setTimeout(() => {
```

Comment:
Why is this needed? And in case it's really needed, is not enough to use `setImmediate()` instead?